packages/core/database/lib/entity-manager/__tests__/relations-orderer.test.js

```diff
@@ -121,6 +121,28 @@ describe('Given I have some relations in the database', () => {
       ]);
     });
   });
+
+  describe('When you connect a relation before one with null order', () => {
+    test('Then it is placed at the end with the correct order', () => {
```

Comment:
The logic makes sense to me but it doesn't seem to match this test description.

id 4 is placed before id 3 and not at the end of the order. am I missing something?